### DISCUSSION TITLE
My Final Year Project as Web Based DAW

### DISCUSSION THREAD

**Discussion by Meesam-12321** [gridsound/daw#99] [General]
Hi guys! I am planning to develop an online DAW, since you guys are aware that most of such products are desktop based applications and my peers are raising questions on its feasibility on the web, because most of the plugins are written using c++ and there can be performance bottlenecks for a MERN stack based DAW, What do you think, should I give it a go?? Actually, if it was a hobby project then i wouldnt have thought much and just started without having feasibility concerns but since it is gonna be my FYP, I cant take chances. Because if I get stuck in an issue in the middle of development , then it would be wastage of alot of time and starting again would make it harder for me to meet the deadlines. What techstack do you guys propose?

**Comment by oix7** [comment]
Don't waste your time tbh.

**Comment by jacob-oreilly** [comment]
I don't have enough experience to probably give real good feedback but if you do go for it check out Rust Wasm. It might allow you to more easily work with the low level stuff in a web format. Another thought would be to use the Web Audio API and see if you'd be able to leverage that to build out a DAW.
